lib/commands.js

```diff
@@ -628,6 +629,47 @@ class Commands {
             inferredConcertoJsonModel.models[0]
         );
     }
+
+    /**
+     * Decorate a given model with given list of dcs and vocab files and print the result
+     * @param {string} modelFile - the model to which vocab and decorator has to be applied
+     * @param {string[]} dcsFiles - the decorator files to be applied to model
+     * @param {string[]} vocFiles - the vocab files to be applied to model
+     * @param {object} options - optional parameters
+     * @returns {string} - Depending on the options, CTO string or JSON string
+     */
+    static async decorate(modelFile, dcsFiles,vocFiles,options) {
+        try {
+            const modelContent = fs.readFileSync(path.resolve(modelFile), 'utf-8');
+            const allDCSFiles = dcsFiles ? dcsFiles.map(file => fs.readFileSync(file, 'utf8')) : [];
+            const allVocsFiles = vocFiles ? vocFiles.map(file => fs.readFileSync(file, 'utf8')) : [];
+            const modelManager = new ModelManager();
+            modelManager.addModel(modelContent);
+            let updatedModelManager = modelManager;
+            if (allDCSFiles.length > 0) {
```

Comment:
Given that you have the nice ternary initialisers above, I think that this condition is redundant. i.e. doing a `.forEach` over an empty array is perfectly fine.